jdi-light-material-ui-tests/src/main/java/io/github/com/pages/displaydata/InsetDividerPage.java

```diff
@@ -1,12 +1,10 @@
 package io.github.com.pages.displaydata;
 
-import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.composite.WebPage;
 import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
 import com.epam.jdi.light.material.elements.displaydata.Divider;
 
-public class InsetDividerFrame extends Section {
-
+public class InsetDividerPage extends WebPage {
```

Comment:
make this class called DivadersPage and put here all locators
VerticalDividerPage can be deleted